packages/datadog-instrumentations/src/http/client.js

```diff
@@ -49,7 +49,7 @@ function patch (http, methodName) {
       const asyncResource = new AsyncResource('bound-anonymous-fn')
 
       return asyncResource.runInAsyncScope(() => {
-        startClientCh.publish({ args, http })
+        startClientCh.publish({ args, http, originalArgs: arguments })
```

Comment:
To be able to detect that the arguments are coming from the request, we need the values that the customer is using in the code, and not the normalized `args` object created by the tracer.